locales/en/yourloops.json

```diff
@@ -60,6 +60,8 @@
   "button-close": "Close",
   "button-create-a-team": "Create a team",
   "button-create-team": "Create team",
+  "care-team-settings": "Care Team settings",
+  "care-team-membership": "Member of these care teams :",
```

Comment:
It shouldn't be plurial because it represents a membership. It means "you are member of theses care teams". Maybe this sentence is confusing, may find antoher more explicit ?

Reply:
Ah yes I did not understand it like that. I don't mind that much, but you can see with Mathieu if you want to be sure :slightly_smiling_face:

Reply:
changed with "Teams I belong to"